client/app/config/gettext.config.js

```diff
@@ -10,6 +10,7 @@
     gettextCatalog.debug = false;
 
     gettextCatalog.loadAndSet = function(lang) {
+      lang = lang.replace('_','-');
```

Comment:
`lang` will be `null` when used with older version of the API .. might either want to remove that `if (lang)` if we don't support it anymore, or do the replace only when not null :).

Also, not that replace replaces just one occurence, not all of them.. (But still no idea what the API would send for Han simplified, so maybe we'll end up needing a lookup table eventually anyway.)